test/module/irohad/ametsuchi/postgres_query_executor_test.cpp

```diff
@@ -674,7 +674,48 @@ namespace iroha {
         : public GetTransactionsExecutorTest {
      protected:
       using Impl = QueryTxPaginationTest;
-
+      
+      void commitTransactionsBlock(std::vector<shared_model::proto::Transaction> txs, size_t height){
```

Comment:
Despite the fact that the code is in tests I suggest to use const reference if possible